components/compliance-service/cmd/inspec_runner/inspec_runner.go

```diff
@@ -40,26 +42,31 @@ func main() {
 func changeToUser(username string) error {
 	user, err := user.Lookup(username)
 	if err != nil {
+		logrus.Errorf("inspec_runner unable to look up user %s: %v", username, err)
```

Comment:
i considered only adding a log statement at the function caller above instead of one per possible error in this function, but given that we're doing this bc it's very hard to debug and that function should be throwing a fatal log on error, i decided to lean towards more verbose/specific error msgs

Reply:
I think we can still have a specific error message here by wrapping the error before we return it.  The caller calls log.Fatal on the error so any information we put into the wrapped error should get logged.